README.md

```diff
@@ -1,12 +1,12 @@
 # Meteora Constant Product AMM SDK (DAMM V2 SDK)
 
-A TypeScript SDK for interacting with the Dynamic CP-AMM on Meteora
+A TypeScript SDK for interacting with the DAMM V2 on Meteora
 
 ## Overview
 
-This SDK provides a set of tools and methods to interact with the [Meteora Dynamic CP-AMM](https://github.com/MeteoraAg/cp-amm). It simplifies common operations like creating pools, managing positions, adding/removing liquidity, swapping tokens, and claiming rewards.
+This SDK provides a set of tools and methods to interact with the [Meteora DAMM-V2](https://github.com/MeteoraAg/cp-amm). It simplifies common operations like creating pools, managing positions, adding/removing liquidity, swapping tokens, and claiming rewards.
```

Comment:
should be (https://github.com/MeteoraAg/damm-v2

Reply:
fixed